src/commands/cmd_query.c

```diff
@@ -58,6 +58,8 @@ void Graph_Query(void *args) {
 	CommandCtx *command_ctx = (CommandCtx *)args;
 	RedisModuleCtx *ctx = CommandCtx_GetRedisCtx(command_ctx);
 	GraphContext *gc = CommandCtx_GetGraphContext(command_ctx);
+
+	CommandCtx_TrackCtx(command_ctx);
```

Comment:
I think it makes more sense to place this in `CommandCtx_New`, as the matching untrack is `CommandCtx_Free`. Otherwise, this must be added at all places that instantiate a `CommandCtx`.

Reply:
Add this call to `PROFILE` and `EXPLAIN`.

Reply:
Also `SLOWLOG` and `BULK`, though `DELETE` doesn't require modification.